### PR TITLE
Blog: New Blog article on Product Roadmaps using Azure Devops

### PR DESCRIPTION
**Pull request by swathynair19**
## What

Added a new md file for a blog post on Product Roadmaps on Azure DevOps

- [x] Added 4 image files to go with the blog in the images folder

# Checklist

- [x] My code follows the style guidelines of this project and is properly formatted
- [x] I have performed a self-review of my own changes

### CODE REVIEW

File: content/blog/product-roadmap-with-azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- Azure DevOps roadmap feature is not necessary if a product has individual teams with their own roadmaps.
+
+- Azure DevOps can be configured in a way that stakeholders can create their own roadmaps using queries, tags, and proper definitions of features and associating respective stories.  
+<a name="_412iyj325u33"></a>Step-By-Step Guide On How To Define Product Roadmaps on Azure DevOps
+
+------------------------------------------------------------------------------------------------
+- Create Features with a common tag. 
+  This will help us as an initial filter in queries to get the team’s related features. 
+
+![Screenshot of Azure DevOps Product Roadmap feature](/images/2023/02/product-roadmap.001.png)
```
Comment (2KAbhishek) [inline content/blog/product-roadmap-with-azure-devops.md:72]: Image loading syntax is incorrect, use this instead:

```
{{< figure src="images/2023/02/product-roadmap.001.png" >}}

```

Same for other screenshots

**Review by 2KAbhishek — CHANGES_REQUESTED** [review]
The red highlighting to hide sensitive data on the screenshots doesn't look good, can we instead try blurring those areas instead

File: content/blog/product-roadmap-with-azure-devops.md (hunk 1 above)
Comment (swathynair19) [reply content/blog/product-roadmap-with-azure-devops.md]: fixed this issue.

**Review by 2KAbhishek — APPROVED** [review]

**Comment by 2KAbhishek** [comment]
@swathynair19 are you able to look at the previews locally after making changes? if not we should do that and verify the blog preview before releasing

**Pull request merged**